docs/creating_your_tooltip.md

```diff
@@ -0,0 +1,105 @@
+# Creating Your Tooltip
+
+## Step 1. Add The Library
+
+We recommend installing the tooltip library and its dependencies via [`npm`](https://www.npmjs.com/):
+
+```bash
+npm install vega-lite-tooltip
+```
+
+The tooltip library contains a `js` file and a `css` file:
+
+```
+node_modules/vega-lite-tooltip/vl-tooltip.min.js
+node_modules/vega-lite-tooltip/vl-tooltip.css
+```
+
+Alternatively, if you want to manually include the tooltip library and its dependencies, you can add the following lines to your HTML `<head>`. Vega-Lite Tooltip depends on [`d3`](https://d3js.org/), [`vega`](https://vega.github.io/vega/), [`vega-lite`](https://vega.github.io/vega-lite/), [`vega-embed`](https://github.com/vega/vega-embed) and [`datalib`](http://vega.github.io/datalib/).
+
+```html
+<!-- Dependencies -->
+<script src="https://d3js.org/d3.v3.min.js"></script>
+<script src="https://vega.github.io/vega/vega.min.js"></script>
+<script src="https://vega.github.io/vega-lite/vega-lite.min.js"></script>
+<script src="https://vega.github.io/vega-editor/vendor/vega-embed.min.js"></script>
+<script src="https://vega.github.io/datalib/datalib.min.js"></script>
+
+<!-- Vega-Lite Tooltip -->
+<script src="https://vega.github.io/vega-lite-tooltip/vl-tooltip.min.js"></script>
+<link rel="stylesheet" type="text/css" href="https://vega.github.io/vega-lite-tooltip/vl-tooltip.css">
+```
+<br>
+
+
+## Step 2. Create A Visualization
+
+In your HTML `<body>`, create a placeholder for your visualization. Give the placeholder a unique `id`, which you can later refer to in your JavaScript. For example:
+
+```html
+<!-- Placeholder for my scatter plot -->
+<div id="vis-scatter"></div>
+```
+
+For [Vega-Lite](https://vega.github.io/vega-lite/): <br>
```

Comment:
hmmm... I kind of want a little more space between steps. So I added a <br> after each step.